examples/api/test/widgets/magnifier/cupertino_magnifier.0_test.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_api_samples/widgets/magnifier/cupertino_magnifier.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('${example.CupertinoMagnifierApp} tests', () {
```

Comment:
For consistency with other examples tests, we don't use group as there are few tests in those test files.